docs/platforms/apple/common/configuration/metric-kit.mdx

```diff
@@ -34,3 +34,23 @@ SentrySDK.start { options in
     options.enableMetricKit = YES;
 }];
 ```
+
+You can enable `enableMetricKitRawPayload` to view the raw MetricKit diagnostic payload in JSON format as an attachment on the converted event in Sentry. This feature is available on Cocoa 8.29.0 and up.
+
+```swift {tabTitle:Swift}
+import Sentry
+
+SentrySDK.start { options in
+    options.dsn = "___PUBLIC_DSN___"
+    options.enableMetricKitRawPayload = true
```

Comment:
```suggestion
    options.dsn = "___PUBLIC_DSN___"
    options.enableMetricKit = true
    options.enableMetricKitRawPayload = true
```

Reply:
IMO it makes sense to keep the line to enable metric kit `enableMetrickit = true` because I assume it's a prerequisite for `enableMetricKitRawPayload`.